sycl/include/sycl/ext/intel/experimental/esimd/memory.hpp

```diff
@@ -750,20 +750,20 @@ lsc_gather(const T *p, __ESIMD_NS::simd<Toffset, N> offsets,
   return detail::lsc_format_ret<T>(Result);
 }
 
-template <
-    typename T, int NElts = 1, lsc_data_size DS = lsc_data_size::default_size,
-    cache_hint L1H = cache_hint::none, cache_hint L3H = cache_hint::none, int N,
-    typename Toffset, typename RegionTy = __ESIMD_NS::region1d_t<Toffset, N, 1>>
+template <typename T, int NElts = 1,
+          lsc_data_size DS = lsc_data_size::default_size,
+          cache_hint L1H = cache_hint::none, cache_hint L3H = cache_hint::none,
+          int N, typename Toffset, typename RegionTy>
```

Comment:
Same comment here. Please rename for better read-ability: `Toffset` -> `OffsetObjT`

Reply:
Ough..., You did it for everything... :)
It was mean to be done for 1st template operand of `simd_view` only, where the 1st tparam is object. 
It should not be applied to `simd`, where the 1st operand is scalar int type.